sections/futures/PositionButtons/PositionButtons.tsx

```diff
@@ -66,58 +67,33 @@ const StyledPositionButton = styled(Button)<PositionButtonProps>`
 		position: relative;
 	}
 
-	${(props) =>
-		props.$position === PositionSide.LONG &&
-		css`
-			color: ${props.theme.colors.selectedTheme.green};
-			${props.$isActive &&
-			css`
-				border: 1px solid ${props.theme.colors.selectedTheme.green};
-				border-radius: 8px;
-				background: linear-gradient(
-					180deg,
-					rgba(127, 212, 130, 0.15) 0%,
-					rgba(71, 122, 73, 0.05) 100%
-				);
-				box-shadow: rgb(127 212 130 / 50%) 0px 0 3px;
+	${(props) => css`
+		font-family: ${props.theme.fonts.bold};
+		color: ${props.theme.colors.selectedTheme.newTheme.button.position[props.$position].color};
+		background: ${props.theme.colors.selectedTheme.newTheme.button.position.background};
```

Comment:
Will we need to replace this `newTheme` thing at some point? Also, maybe not your theming but this red looks really aggressively bright.

Reply:
Yeah. We're going to get rid of it. The only reason I'm leaving it in for now is that I want to make sure this PR doesn't break anything.

Reply:
Also, I think your point about the red color is right. I'll reach out to Yashar about it.